src/services/user-map-service.js

```diff
@@ -40,9 +40,9 @@ export const initMap = (hostId) => {
   userMap.addLayer(tiles);
 };
 
-export const addMapMarker = ({ username, lat, lng, caption }) => {
+export const addMapMarker = ({ username, lat, lng, caption, hideProfilePicture }) => {
   const mark = marker([lat, lng], {
-    icon: username ? userIcon(username) : defaultIcon,
+    icon: (username && hideProfilePicture !== false) ? userIcon(username) : defaultIcon,
```

Comment:
I think this can be `icon: (username && !hideProfilePicture) ? userIcon(username) : defaultIcon,`